### ISSUE TITLE
tasktracker updates

### ISSUE THREAD

**Issue by brianwdavis** [precision-sustainable-ag/tech-dashboard#879]
For the decomp section, show time zero (all categories) when `protocols.in_field_biomass` = 1; only show times 1-5 (all categories) if `protocols.decomp_biomass` = 1.

To clarify:

- For decomp-empty weights and decomp-fresh weights, only show at any time if `protocols.decomp_biomass` = 1.
- For decomp-recovery date and decomp-dry weights, show at time 0 if `protocols.in_field_biomass` = 1 (should be all sites).  Do not show the site at all for times 1-5 if `protocols.decomp_biomass`  does not = 1.

**Comment by brianwdavis** [comment]
```
[
    {
        "code": "EDF",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "HNZ",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "IFC",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "KIZ",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 0,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "0"
    },
    {
        "code": "LRH",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "0"
    },
    {
        "code": "SHX",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "TOX",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "URF",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 0,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "0"
    },
    {
        "code": "VZR",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    },
    {
        "code": "XZY",
        "count": "4",
        "protocols": {
            "sensor_data": 1,
            "bulk_density": 0,
            "corn_disease": null,
            "farm_history": 1,
            "soil_texture": 1,
            "gps_locations": 1,
            "soil_nitrogen": 0,
            "yield_monitor": null,
            "decomp_biomass": 1,
            "cash_crop_yield": 1,
            "in_field_biomass": 1,
            "weed_visual_rating": null,
            "weed_quadrat_photos_beta": 0
        },
        "flag": "1"
    }
]
```

**Comment by brianwdavis** [comment]
Fixed in #887
